pkg/integrations/cloudwatch_exporter/cloudwatch.go

```diff
@@ -0,0 +1,106 @@
+package cloudwatch_exporter
+
+import (
+	"fmt"
+	"github.com/grafana/agent/pkg/integrations"
+	integrations_v2 "github.com/grafana/agent/pkg/integrations/v2"
+	"github.com/grafana/agent/pkg/integrations/v2/metricsutils"
+	"time"
+
+	"github.com/go-kit/log"
+)
+
+func init() {
+	integrations.RegisterIntegration(&Config{})
+	integrations_v2.RegisterLegacy(&Config{}, integrations_v2.TypeMultiplex, metricsutils.NewNamedShim("cloudwatch_exporter"))
```

Comment:
```suggestion
	integrations_v2.RegisterLegacy(&Config{}, integrations_v2.TypeMultiplex, metricsutils.NewNamedShim("cloudwatch"))
```
v2 integrations appear to have dropped the `_exporter` suffix (there's a test which enforces this)

Reply:
👍